crowdstrikeoauthapi_connector.py

```diff
@@ -753,17 +762,36 @@ def _handle_list_incidents(self, param):
         action_result = self.add_action_result(ActionResult(dict(param)))
 
         max_limit = None
-        sort_data = ["assigned_to.asc", "assigned_to.desc", "assigned_to_name.asc", "assigned_to_name.desc", "end.asc", "end.desc",
-            "modified_timestamp.asc", "modified_timestamp.desc", "name.asc", "name.desc", "sort_score.asc", "sort_score.desc",
-            "start.asc", "start.desc", "state.asc", "state.desc", "status.asc", "status.desc"]
-
-        resp = self._check_data(action_result, param, max_limit, sort_data)
+        sort_data = [
+            "assigned_to.asc",
+            "assigned_to.desc",
+            "assigned_to_name.asc",
+            "assigned_to_name.desc",
+            "end.asc",
+            "end.desc",
+            "modified_timestamp.asc",
+            "modified_timestamp.desc",
+            "name.asc",
+            "name.desc",
+            "sort_score.asc",
+            "sort_score.desc",
+            "start.asc",
+            "start.desc",
+            "state.asc",
+            "state.desc",
+            "status.asc",
+            "status.desc",
+        ]
+
+        params = {k: param[k] for k in param.keys() if k in ["filter", "limit", "offset", "sort"]}
```

Comment:
Maybe it is worth to move `["filter", "limit", "offset", "sort"]` to consts, as it is used in multiple places?